modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java

```diff
@@ -779,6 +787,20 @@ private String _getValueString(
 		return jsonObject.toString();
 	}
 
+	private String _sanitize(String ddmFormFieldType, String value)
+		throws SanitizerException {
+
+		if (Objects.equals(
+				ddmFormFieldType, DDMFormFieldTypeConstants.RICH_TEXT)) {
+
+			return SanitizerUtil.sanitize(
+				0, 0, 0, DDMFieldAttribute.class.getName(), 0,
+				ContentTypes.TEXT_HTML, value);
+		}
+
+		return value;
+	}
```

Comment:
@henriquelimadotcom Instead of doing this in this class, can we change to use the `HTMLSanitizerDDMFormFieldValueTransformer`, in the [DefaultDDMStorageAdapter](https://github.com/liferay/liferay-portal/blob/2d5b39b2583dc4cc5c7dbe8f55d7cc074d7ce17f/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/DefaultDDMStorageAdapter.java#L66-L67) class, like in [here](https://github.com/liferay/liferay-portal/blob/2d5b39b2583dc4cc5c7dbe8f55d7cc074d7ce17f/modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/storage/BaseStorageAdapter.java#L133-L148)?

You can use the `DDMStorageAdapterSaveRequest` to get the `groupId` and `userId`, for example..